src/EncompassRest/Loans/LoanEntity.cs

```diff
@@ -99,7 +98,6 @@ public enum LoanEntity
         LockDenialLog = 95,
         LockRequestBorrower = 96,
         LockRequestLog = 97,
-        LOCompensation = 98,
```

Comment:
why is this dropping off?

Reply:
Because LOCompensation is not actually a loan entity type. When retrieving the loan schema for LOCompensation you get the ElliLOCompensation entity's schema.

Reply:
See https://forum.elliemae.com/index.php?/topic/6271-loan-entity-schema-issues/

Reply:
so, should we have a value for this of elilocomp? and keep the schema entity match?

Reply:
ElliLOCompensation is already there.